### ISSUE TITLE
cOALA - Intentional or Typo?

### ISSUE THREAD

**Issue by peterennis** [coala/coala#5249]
![capture246](https://user-images.githubusercontent.com/140737/37239316-35c8a9c0-23ee-11e8-8d92-72e99d3de580.png)

**Comment by Makman2** [comment]
Intentional, because coala is written always with a lower-case "c" ;D However I agree that it's arguable and looks a bit weird ;)

**Comment by sangamcse** [comment]
I think it is more eye catcher. It's good actually until you are a grammar teacher. 😉

**Comment by nvzard** [comment]
What If we do this? https://prnt.sc/ipqad7
all in lowercase?

**Comment by Makman2** [comment]
Not a bad idea :+1:

**Comment by nvzard** [comment]
Yeah, I thinks this https://prnt.sc/ipqaz5 is looking pretty neat ;)
